units/user@.service.in

```diff
@@ -22,6 +22,7 @@ ExecStart={{ROOTLIBEXECDIR}}/systemd --user
 Slice=user-%i.slice
 KillMode=mixed
 Delegate=pids memory cpu
+DelegateSubControlGroup=init.scope
```

Comment:
udev's create_subcgroup() logic should be reworked to make use of this too, btw.